route/journal-router.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const Journal = require('../model/journal.js');
```

Comment:
Custom error statuses and messages help with testing when you want errors to return the appropriate status.